op-deployer/pkg/deployer/bootstrap/superchain.go

```diff
@@ -113,15 +113,24 @@ func SuperchainCLI(cliCtx *cli.Context) error {
 		Paused:                    paused,
 	}
 
-	if err := cfg.RequiredProtocolVersion.UnmarshalText([]byte(requiredVersionStr)); err != nil {
-		return fmt.Errorf("failed to parse required protocol version: %w", err)
+	// Default to op-geth params.OPStackSupport if not specified for required and recommended protocolversions
+	if requiredVersionStr != "" {
```

Comment:
Is this logic correct? As I read this it looks like this will always set the versions to `params.OPStackSupport` as long as there is no error and the user specified a required/recommended version.

Reply:
Double-checked. The logic is as I intended:
* if the user specified a `requiredVersion`, it will use that
* else, use `params.OPStackSupport`

Reply:
Are you sure? I think the boolean is flipped. The boolean is checking for `requiredVersionStr != ""`, which will be the case if the user _sets_ a `requiredVersion`.

Reply:
Ah never mind, I missed the indentation - this works.